Automattic/wp-calypso
# Plugins: Add sites plugins endpoint.

#### Changes proposed in this Pull Request

This PR implements the new /me/sites/plugins endpoint. See D76277-code

The idea here is that we get the data from .com cache site ( like the current /me/sites ) endpoint so that we don't have to make requests for each individual site which can take a really long time if you have many jetpack sites. 

#### Testing instructions

* Apply the following patch to your sandbox. 
* Visit /plugins/manage and notice that things load as expected. Compare your results to wordpress.com/plugins/manage
